Analytical_TTL Replace Functionality (#25839)
Azure/azure-sdk-for-python: sdk/cosmos/azure-cosmos/CHANGELOG.md

#### Features Added
- GA release of integrated cache functionality. For more information on integrated cache please see [Azure Cosmos DB integrated cache](https://docs.microsoft.com/azure/cosmos-db/integrated-cache).

> - GA release of analytical ttl replace functionality. For more information on analytical ttl please see [Azure Cosmos DB analytical store](https://docs.microsoft.com/azure/cosmos-db/analytical-store-introduction).

Reviewer: no need to mark this as GA - GA is when we release functionality that was previously in beta. 
```suggestion
- Added ability to replace analytical ttl on containers. For more information on analytical ttl please see [Azure Cosmos DB analytical store](https://docs.microsoft.com/azure/cosmos-db/analytical-store-introduction).
```

Author: got ya
